strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java

```diff
@@ -387,7 +391,80 @@ private void getPackageInternal(String storageId,
             response.setStatus(INTERNAL_SERVER_ERROR.value());
         }
     }
+    
+    @ApiOperation(value = "Used to download the headers for a package")
+    @ApiResponses(value = { @ApiResponse(code = HttpURLConnection.HTTP_OK, message = "The package was downloaded successfully."),
+                            @ApiResponse(code = HttpURLConnection.HTTP_INTERNAL_ERROR, message = "An error occurred.") })
+    @PreAuthorize("hasAuthority('ARTIFACTS_RESOLVE')")
+    @RequestMapping(path = "{storageId}/{repositoryId}/{commandName:(?:download|package)}/{packageId}/{packageVersion}", method = RequestMethod.HEAD, produces = MediaType.APPLICATION_OCTET_STREAM)
+    public void downloadPackageHeaders(@ApiParam(value = "The storageId", required = true) @PathVariable(name = "storageId") String storageId,
```

Comment:
@sanket407 , @sbespalov : 

I may have asked this already (or thought of asking it), but why do we need these two identical methods:
* `public void downloadPackageHeaders(String storageId, String repositoryId, String packageId, String packageVersion, HttpServletResponse response)`
* `public void getPackageHeaders(String storageId, String repositoryId, String packageId, String packageVersion, HttpServletResponse response)`

Reply:
@carlspring  I think @sbespalov  had said both are necessary last time .

Reply:
@carlspring to be honest I don't remember exact Nuget cases when each of them used but they have different URL (so there is different MVC declarations) format but logic is the same (so they delegate invocation into one internal method).